tests/installation/boot_into_snapshot.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+###  this test test the read-only snapshot, from console. (1)
+###  we print some console output only for nice debugging (1b)
+###  if variable UPGRADE is set, we want to check that the 2 snapshots
+###  before_upgrade and after upgrade are not identical. test is made
+###  by checking the /etc/os-release, this is compatible also for openSUSE-TW (2)
+
+use strict;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    wait_idle;
```

Comment:
was for ensure that i don't have i still screen before logging to console. Just double check for security